mender-convert-modify

```diff
@@ -277,6 +222,61 @@ ${data_part_device}   /data          auto       ${MENDER_DATA_PART_FSTAB_OPTS}
 $fstab
 EOF
 
+log_info "Installing Mender client and related files"
+
+if [ "${MENDER_CLIENT_INSTALL}" = "y" ]; then
+    case "${MENDER_CLIENT_VERSION}" in
+        1.* | 2.* | 3.*)
+            log_info "Installing Mender client version ${MENDER_CLIENT_VERSION}"
+            deb_get_and_install_package mender-client mender-client "${MENDER_CLIENT_VERSION}"
+            mender_client_deb_name="${DEB_NAME}"
+            ;;
+        *)
+            # For 4.x, master and latest(*) install the new packages
+            # At the time of writing, this will be broken for "latest" until mender-client v4 is
+            # released. Having an special handling deemed to complex for the short time benefit.
+            log_info "Installing Mender Auth and Mender Update version ${MENDER_CLIENT_VERSION}"
```

Comment:
S: This could be logged in the `deb_get_and_install_package` function

Reply:
I think it should be logged here, because the deb installer logging is much more detailed and low level, with a full URL. Which is also useful, but much harder to read. The message right here is the message I was looking for myself when I was watching the log go by.